### PR TITLE
Minutes for 10/07/2016

### PR DESCRIPTION
**Pull request by ChadH1971**
Last, but not least, the minutes for today.

### CODE REVIEW

**Comment by asterwolf** [comment]
I think they look great. Not a thing in which I would suggest to change. I'll wait it out for @pathawks to view. He can spot a missing semi-colon from a mile away. ;)

**Comment by ChadH1971** [comment]
He admitted to being a Slackbot

File: _posts/minutes/2016-10-07-minutes.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
+[LISTEN TO THIS MEETING](https://fccdl.in/iPvoj2r1R)
```
Comment (pathawks) [inline _posts/minutes/2016-10-07-minutes.markdown]: Is this the best way to display this?

Comment (asterwolf) [inline _posts/minutes/2016-10-07-minutes.markdown]: Agreed. It feels out of place as well. Maybe move toward the end?

Comment (ChadH1971) [inline _posts/minutes/2016-10-07-minutes.markdown]: What other way would you suggest? It's an external link. Is just positioning or elevating it with an h tag? Moving it to somewhere else in the document?

**Comment by ChadH1971** [comment]
I made changes to the layout, you can see my commit comment.

File: _posts/minutes/2016-10-07-minutes.markdown [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[LISTEN TO THIS MEETING](https://fccdl.in/iPvoj2r1R)
+&nbsp;&nbsp;&nbsp;Listen to this meeting here: [here](https://fccdl.in/iPvoj2r1R)
```
Comment (pathawks) [inline _posts/minutes/2016-10-07-minutes.markdown]: What's with all the `&nbsp;`?

Comment (asterwolf) [inline _posts/minutes/2016-10-07-minutes.markdown]: I was indenting the line to make things line up. Best way I knew how. :p

Comment (asterwolf) [inline _posts/minutes/2016-10-07-minutes.markdown]: screen shot 2016-10-11 at 4 24 57 pm

The ordered list.

Comment (asterwolf) [inline _posts/minutes/2016-10-07-minutes.markdown]: Hey @pathawks, did everything look okay with this approach? I just picked something simple so we could get these meetings minutes up.

Comment (asterwolf) [inline _posts/minutes/2016-10-07-minutes.markdown]: Well sir, I would like to know how I can display it better. 😝 . I thought it was a step up from what it was before. 😆

**Pull request merged**